packages/eslint-config-airbnb/package.json

```
@@ -68,7 +68,7 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^2.3.0 || ^1.7.0",
+    "eslint-plugin-react-hooks": "^4.0.0 || ^3.0.0 || ^2.3.0 || ^1.7.0",
```

```suggestion
    "eslint-plugin-react-hooks": "^4 || ^3 || ^2.3.0 || ^1.7.0",
```
the same line must be duplicated in peerDependencies - otherwise you haven't actually allowed anything :-)

---

Oops, that's right 😅 Thank you for correcting my mistake!